msteams-platform/tabs/how-to/authentication/auth-tab-aad.md

```diff
@@ -48,14 +48,14 @@ Trigger the authentication flow by a user action. Avoid opening the authenticati
 
 Add a button to your configuration or content page to enable the user to sign in when needed. This can be done in the tab [configuration](~/tabs/how-to/create-tab-pages/configuration-page.md) page or any [content](~/tabs/how-to/create-tab-pages/content-page.md) page.
 
-Azure AD, like most identity providers, doesn't allow its content to be placed in an `iframe`. This means that you'll need to add a pop-up page to host the identity provider. In the following example, this page is `/tab-auth/simple-start`. Use the `authentication.authenticate()` function of the TeamsJS library to launch this page when the button is selected.
+Azure AD, similar to most identity providers, doesn't allow its content to be placed in an `iframe`. This means you need to add a page to host the identity provider that is displayed inside a pop-up window by Teams client. In the following example, the page is `/tab-auth/simple-start`. Use the `authentication.authenticate()` function of the TeamsJS library to launch this page when the button is selected.
```

Comment:
Azure AD, similar to most identity providers, doesn't allow its content from being embedded in an `iframe`. However, you must incorporate a page to host the identity provider, which Teams client displays within a pop-up window. In the following example, the designated page is `/tab-auth/simple-start`. Utilize the `authentication.authenticate()` function from the TeamsJS library to launch this page after the button is selected.

Reply:
Updated.